src/jsxc.lib.xmpp.chatState.js

```diff
@@ -130,9 +130,10 @@ jsxc.xmpp.chatState.onPaused = function(ev, jid) {
    }
 
    if (usersComposing.length === 0) {
-      var duration = parseFloat(el.css('transition-duration'));
+      var durationValue = el.css('transition-duration') || '0';
```

Comment:
I think it doesn't matter. `0s` would be maybe cleaner, but the result is the same :smile: